NOTICE.txt

```diff
@@ -0,0 +1,996 @@
+NOTICES AND INFORMATION
+Do Not Translate or Localize
+
+This software incorporates material from third parties.
+Microsoft makes certain open source code available at https://3rdpartysource.microsoft.com,
+or you may send a check or money order for US $5.00, including the product name,
+the open source component name, platform, and version number, to:
+
+Source Code Compliance Team
+Microsoft Corporation
+One Microsoft Way
+Redmond, WA 98052
+USA
+
+Notwithstanding any other terms, you may reverse engineer this software to the extent
+required to debug changes to any libraries licensed under the GNU Lesser General Public License.
+
+---------------------------------------------------------
+
+As an OPC Foundation Corporate Member, Microsoft distributes any code dual licensed under the RCL or GPLv2 license, under the RCL license.
+
+---------------------------------------------------------
+
+OPCFoundation.NetStandard.Opc.Ua.Core 1.4.368.53 - OTHER
+

```

Comment:
nitpick :-) -- 369.30

Reply:
Updating the version number in the notice file every time we update the dependency sounds like cumbersome manual work 😅